src/contrast/pages/UploadNewPaper/ExperimentsSection/Stimuli.jsx

```diff
@@ -52,10 +52,10 @@ export default function Stimuli({
   );
 
   useEffect(() => {
```

Comment:
Its not good to manipulate objects inside use effect without a state hook or a ref